### PR TITLE
hey: init at 0.1.3

### PR DESCRIPTION
**Pull request by Br1ght0ne**
###### Motivation for this change
https://github.com/rakyll/hey/releases/tag/v0.1.3

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---
This change is [Reviewable](https://reviewable.io/reviews/nixos/nixpkgs/79281)

### CODE REVIEW

**Review by Frostman — CHANGES_REQUESTED** [review]
`nixpkgs-review` passed ok, binary works correctly.

**Comment by Frostman** [comment]
Change LGTM, verified with `nixpkgs-review`, binary works correctly (NixOS, x86_64).

**Review by Frostman — APPROVED** [review]

File: pkgs/tools/networking/hey/default.nix [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{ buildGoModule, lib, fetchFromGitHub }:
+
+buildGoModule rec {
+    pname = "hey";
```
Comment (yegortimoshenko) [inline pkgs/tools/networking/hey/default.nix]: Indentation seems to be off?

File: pkgs/tools/networking/hey/default.nix [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    };
+
+    modSha256 = "0a00kcyagqczw0vhl8qs2xs1y8myw080y9kjs4qrcmj6kibdy55q";
+
+    meta = with lib; {
+        description = "HTTP load generator, ApacheBench (ab) replacement";
+        homepage = "https://github.com/rakyll/hey";
+        license = licenses.asl20;
+        maintainers = with maintainers; [ filalex77 ];
+        platforms = platforms.all;
```
Comment (yegortimoshenko) [inline pkgs/tools/networking/hey/default.nix]: `buildGoModule` sets default `meta.platforms`: https://github.com/NixOS/nixpkgs/blob/f373ecec8f49b24c3070ab4f66a680bee06980a5/pkgs/development/go-modules/generic/default.nix#L216-217

**Review by lukateras — COMMENTED** [review]
LGTM otherwise!

**Comment by filalex77** [comment]
Thanks, @Frostman and @yegortimoshenko!

**Comment by yegortimoshenko** [comment]
@GrahamcOfBorg build hey

**Review by lukateras — APPROVED** [review]

**Pull request merged**